boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm4.dts

```diff
@@ -64,28 +64,29 @@
 			compatible = "fixed-partitions";
 			#address-cells = <1>;
 			#size-cells = <1>;
-			/* Note: CM4 does not boot from flexspi, but CM7 code will be
-			 * stored in slot0
-			 */
+
 			boot_partition: partition@0 {
 				label = "mcuboot";
-				reg = <0x00000000 DT_SIZE_K(64)>;
+				reg = <0x00000000 DT_SIZE_K(128)>;
 			};
-			slot0_partition: partition@10000 {
+			/* Note slot 0 has one additional sector,
+			 * this is intended for use with the swap move algorithm
+			 */
+			slot0_partition: partition@20000 {
 				label = "image-0";
-				reg = <0x00010000 DT_SIZE_M(3)>;
+				reg = <0x00020000 301000>;
```

Comment:
Should the size value be in HEX i.e `0x301000`

Reply:
Yes, these values should be in HEX, I missed that. Fixed now.